src/main/java/co/cask/plugin/format/MultiTableConf.java

```diff
@@ -48,15 +51,23 @@
 
   @Macro
   @Nullable
-  @Description("A pattern that defines which tables should be read from. " +
-    "Any table whose name matches the pattern will read. If not specified, all tables will be read.")
-  private String tableNamePattern;
```

Comment:
I don't think we should remove tableNamePattern (backwards incompatible change). We can just add schemaPattern.

Reply:
Updated.